content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -56,14 +56,14 @@ To execute the code examples provided in this article you should have each of th
 If you don't have an Azure account, you can create one for free here:
 <https://portal.azure.com>
 
-## **What is Azure Key Vault?**
+## What is Azure Key Vault?
 
 Azure Key Vault is a cloud service that enables you to store your application's secrets in a safe and secure place. You can keep the keys, passwords, certificates, and other secrets safely and securely. Azure Key Vault is a service that lets you store secrets and other sensitive configuration data for a particular application. You can take advantage of Azure Key Vault to set parameters shared among multiple applications, including applications running in App Service.
 
 It enables you to isolate the sensitive and non-sensitive data in your application. For example, you can use application settings to store default parameters or key-value pairs containing some default settings used by the
 application. On the contrary, you can use Azure Key Vault to store API keys, secret keys, database connection strings, or Client IDs used in your application.
 
-## **Understanding Managed Identity**
+## Understanding Managed Identity
 
 Managed identity is a concept that eliminates the need of having to store credentials once an application has been deployed to the cloud. By using managed identity, we can securely access a variety of Azure services without having to store any credentials like as connection strings or passwords. Managed identity may be used to connect to Key Vault from an Azure Function App or an Azure Web App, as well as to connect to Azure Blob Storage from an Azure Web App.
```

Comment:
Changes: Managed identity is a concept that eliminates the need to store credentials once an application has been deployed to the cloud. (deleted `of having`)

The second highlighted part is repeating the same fact stated in the previous sentence: ``` it helps ``` (*this part is edited*) securely access a variety of Azure services `without having to store any credentials like as connection strings or passwords`.

Changes: `You can also use m`anaged identity to connect to Key Vault from an Azure Function App or an Azure Web App, as well as to connect to Azure Blob Storage from an Azure Web App.

Changes: Managed identities are two types: (deleted  `of the following`)